tasks/managed/collect-push-rpm-params/collect-push-rpm-params.yaml

```diff
@@ -0,0 +1,46 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: collect-push-rpm-params
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: push-koji
+spec:
+  params:
+    - name: dataPath
+      type: string
+      description: |
+        Path to the JSON file of the merged data to use in the data workspace.
+  steps:
+    - name: collect-push-rpm-params
+      image: quay.io/redhat-appstudio/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
```

Comment:
this has the wrong org

Reply:
hmm, oh, sorry my bad. I was konflux-ci before.